portiaui/app/components/inspector-panel.js

```diff
@@ -45,6 +45,26 @@ export function getAttributeList(element) {
     return attributeList;
 }
 
+export function getDefaultAttribute(element) {
+    // TODO: Remove attributes that have been already annotated
```

Comment:
True, not sure what the most common use case would be, if annotating the same attribute twice, or annotating different attributes. There are few cases where you want different attributes (href and content, src and title) but also few use cases for annotating the same attribute twicd.

At the moment when an element is already annotated the default action with the magic tool enabled will be to select the existing annotation anyway